dashboard/app/controllers/plc/user_course_enrollments_controller.rb

```diff
@@ -0,0 +1,55 @@
+class Plc::UserCourseEnrollmentsController < ApplicationController
+  #TODO: Why do I need to have the except here?
+  load_and_authorize_resource except: [:create]
```

Comment:
Per our discussion, let's have `load_and_authorize_resource` handle loading `@course` and update the views.

(Same feedback for the other controllers.)